### ISSUE TITLE
Blanket overcrowds individual volume levels in Gnome Settings

### ISSUE THREAD

**Issue by avila** [rafaelmardojai/blanket#263]
Sometimes I like to use the Sound page of the Gnome Settings to control de volume level per application. It happens that an entry for each sound type is created as soon as they are created in Blanket. They don't disappear after muting it, only after closing the app. 

I don't know if this is manageable from Blanket but the entries of other apps, such as Firefox, are removed as soon as the audio/video is over or the tab closed. In my use case it would be great if Blanket would create only one entry for the overall sound level if possible. Otherwise, removing the muted entries would be an improvement. 

![image](https://user-images.githubusercontent.com/31955632/181006864-81d3421f-91bc-4a4a-a43d-0ffbf1a5331c.png)

It also affects some extensions that control audio output per application, such as [Application Volume Mixer](https://extensions.gnome.org/extension/3499/application-volume-mixer/)

![image](https://user-images.githubusercontent.com/31955632/181008767-37cbf690-2f47-499d-8a40-b06ab98196f8.png)

Of course this is only a minor annoyance and it should likely have low priority. But if it is an easy fix I and maybe users of such extensions would greatly appreciate :)

**Comment by rafaelmardojai** [comment]
Duplicated of  #82.
